src/components/container/BloodPressureVisualization/BloodPressureVisualization.css

```diff
@@ -76,4 +76,10 @@
 .mdhui-blood-pressure-metrics-block .mdhui-blood-pressure-metric-not-normal{
     font-size: 17px;
     color: var(--mdhui-color-danger);
+}
+
+.mdhui-blood-pressure-interval-not-supported{
+    text-align: center;
+    color : var( --mdhui-color-danger);
+    font-size: 16px;
```

Comment:
We should use `em`'s for font size wherever possible.  Our default font size is 17px (I think) so this would be around `0.95em`.